.travis.yml

```diff
@@ -14,14 +14,14 @@ before_install:
     - conda update --all --yes
     - travis_retry conda create --yes -n TEST $CONDA --file requirements.txt
     - source activate TEST
-    - travis_retry conda install --yes --file requirements-dev.txt
+    - travis_retry pip install -r requirements-dev.txt
```

Comment:
I've got pytest-cov in the NOAA-ORR-ERD channel, if that helps.

CHB

On Feb 15, 2016, at 11:10 AM, Filipe <notifications@github.com> wrote:

In .travis.yml
<https://github.com/pyugrid/pyugrid/pull/130#discussion_r52934329>:

> @@ -14,14 +14,14 @@ before_install:
>      - conda update --all --yes
>      - travis_retry conda create --yes -n TEST $CONDA --file requirements.txt
>      - source activate TEST
> -    - travis_retry conda install --yes --file requirements-dev.txt
> +    - travis_retry pip install -r requirements-dev.txt

Note to self: pytest-cov is unavailable in the default channel and outdated
in the IOOS channel. For now it is OK to install the light weight
dependencies (requirements-dev.txt) with pip and the heavy weight (
netCDF4-python, scipy, etc) with conda.

The long term goal is to add those packages to conda-forge and just add
that channel here.

—
Reply to this email directly or view it on GitHub
<https://github.com/pyugrid/pyugrid/pull/130/files#r52934329>.